### ISSUE TITLE
Consider changing used GPG references for the PGP

### ISSUE THREAD

**Issue by jan-kolarik** [rpm-software-management/dnf5#181]
This is intended as a space for discussion whether we want to change mentions of the GPG inside the codebase for PGP or OpenPGP, etc.

This issue is created based on the discussion in the https://github.com/rpm-software-management/dnf5/pull/166.

**Comment by pmatilai** [comment]
Yup. To recoup, GPG is an implementation and so is PGP. When not talking about a particular implementation, OpenPGP is the correct term to use. Goes for signatures, keys, all that.

**Comment by pmatilai** [comment]
While user interfaces can be changed at leisure, it'd be good to fix the API exposure, including config variable naming, before dnf5 goes to widespread usage. Changing it later will be much much more difficult.

**Comment by j-mracek** [comment]
The rename of configuration options is difficult. If it is a configuration option in `.repo` file it is very painful and the benefit is pretty low.

**Comment by decathorpe** [comment]
Would it be possible to change the preferred configuration option name, but keep accepting the old name for backwards compatibility for some period of time? That should make it less painful.

**Comment by evan-goode** [comment]
Much renaming of "PGP" or "GPG" to "OpenPGP" was done in https://github.com/rpm-software-management/dnf5/pull/1931, https://github.com/rpm-software-management/dnf5/pull/2280.

We still have the options `gpg_key`, `repo_gpgcheck`, and `pkg_gpgcheck` which could use aliases. And a couple enum values (`LRO_GPGCHECK`, `ERROR_GPG_CHECK`) that are probably not worth changing at this point.
